.github/workflows/_win-test.yml

```diff
@@ -144,7 +144,7 @@ jobs:
           NUM_TEST_SHARDS: ${{ matrix.num_shards }}
           TEST_CONFIG: ${{ matrix.config }}
           PR_BODY: ${{ github.event.pull_request.body }}
-          TORCH_CUDA_ARCH_LIST: "7.0"
+          TORCH_CUDA_ARCH_LIST: "8.6"
```

Comment:
Looking at https://github.com/search?q=repo%3Apytorch%2Fpytorch%20TORCH_CUDA_ARCH_LIST&type=code, there are a couple places where we default this value to 5.2

Would it make sense to change those defaults as well?

Reply:
Yeah, I can change that.  `5.2` is probably a legacy default that is still part of the code till date.

Reply:
5.2 is incredibly old wouldn't recommend doing that

Reply:
Oh, I mean removing 5.2 as the default as it's indeed very old :)